### PR TITLE
bug in cv format with fullname

### PR DESCRIPTION
**Pull request by nooraangelva**
Ref: https://github.com/cern-sis/issues-inspire/issues/38
error: https://sentry.sis.cern.ch/inspire/hep/issues/36765/?query=is%3Aunresolved

### CODE REVIEW

File: backend/tests/integration/records/serializers/data/2108903.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        "$ref": "https://inspirehep.net/api/authors/2108904"
+      },
+      "full_name": "Brown, Michael",
+      "inspire_roles": [
+        "supervisor"
+      ],
+      "signature_block": "BRANm"
+    }
+  ]
+}
```
Comment (drjova) [inline backend/tests/integration/records/serializers/data/2108903.json:40]: this record has `full_name` on both authors, hence we are not testing the actual corner case.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -1,18 +1,18 @@
 repos:
   - repo: https://github.com/psf/black
-    rev: stable
+    rev: 22.6.0
```
Comment (drjova) [inline .pre-commit-config.yaml:3]: why we need to change the versions? 🤔

File: backend/tests/unit/records/marshmallow/literature/common/test_reference_item.py [hunk 3]
```diff
@@ -241,3 +241,39 @@ def test_returns_arxiv_eprints_from_the_resolved_record(
...
+        "urls": [
+            {"value": "http://www.claymath.org/prize-problems"},
+            {"value": "http://www.arthurjaffe.com"},
+        ],
+        "collaborations": [{"value": "CMS"}],
+        "collaborations_with_suffix": [{"value": "ATLAS Team"}],
+    }
+    record = LiteratureRecord(dump)
+    result = schema.dumps(record).data
+    assert expected == orjson.loads(result)
```
Comment (drjova) [inline backend/tests/unit/records/marshmallow/literature/common/test_reference_item.py:279]: hmm I'm very confused, this test is not related to the code changes, you changed something in the CVSchema :/ Do I miss something?

File: .pre-commit-config.yaml (hunk 2 above)
Comment (nooraangelva) [reply .pre-commit-config.yaml:3]: Sorry, I forgot to change it back. For some reason, if I make changes to python scripts Github doesn't let me commit until I have updated the pre-commit. Usually, afterwards I revert the pre-commit back.

File: backend/tests/unit/records/marshmallow/literature/common/test_reference_item.py (hunk 3 above)
Comment (nooraangelva) [reply backend/tests/unit/records/marshmallow/literature/common/test_reference_item.py:279]: Thank you for pointing that out. I don't know how I managed to get so confused with the codes that I wrote the test in the wrong place.

File: backend/tests/integration/records/serializers/test_cv.py [hunk 4]
```diff
@@ -620,3 +624,72 @@ def test_literature_detail_cv_link_alias_format(inspire_app):
...
+                    {
+                        "value": "LIGO - California Institute of Technology - Pasadena - CA 91125 - USA"
+                    }
+                ],
+            }
+        ],
+        "arxiv_eprints": [{"value": "1606.09129", "categories": ["hep"]}],
+    }
+    record = LiteratureRecord(dump)
+    result = schema.dumps(record).data
```
Comment (drjova) [inline backend/tests/integration/records/serializers/test_cv.py:693]: I would test it like https://github.com/inspirehep/inspirehep/blob/656f88b0742ac384aa5e080019ec8fca8e91b59d/backend/tests/integration/records/serializers/test_cv.py#L18-L36
